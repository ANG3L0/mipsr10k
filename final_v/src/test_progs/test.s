


   
     lda  $r1, 4
     lda  $r2, 2
     addq $r1,0x8,$r3
     addq $r2,0x8,$r4
     call_pal    0x555
