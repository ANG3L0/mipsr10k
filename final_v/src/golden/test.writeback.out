PC=0000000000000000, REG[ 1]=0000000000000004
PC=0000000000000004, REG[ 2]=0000000000000002
PC=0000000000000008, REG[ 3]=000000000000000c
PC=000000000000000c, REG[ 4]=000000000000000a
PC=0000000000000010, ---
